main.py

```diff
@@ -1,3 +1,13 @@
+from typing import Dict, Set
+
+from coin import Coin
+from costumer import Costumer
+from drink import Drink
+from machine import Machine, NoSuchDrinkError
+from resource import Resource
+
+STARTING_COIN = 10
+
 MENU = {
```

Comment:
Thats nice, along with the STARTING_COIN. Perhaps you can export this to a settings INI file or something. Or a config module or something